src/components/quote/quote.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { TextStyles } from '../typography/typography-v2';
+import { theme } from '../layout/theme';
+
+interface QuoteProps {
+  children: ReactNode | ReactNode[];
+}
+
+const QuoteText = styled.p`
+  ${TextStyles.quote}
+  color: ${theme.palette.text.topline};
+  padding-left: 20px;
+  margin: 0;
+`;
+
+const Quote = ({ children }: QuoteProps) => {
+  return <QuoteText>{children}</QuoteText>;
```

Comment:
The design doesn't show any cite sources and such but the this [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote) shows a great example of how to do it with `figure` combined with the blockquote. Maybe we can work on that in a following PR to make this component more useful as I can imagine to use this in blog posts if available.

```
<figure>
    <blockquote cite="https://www.huxley.net/bnw/four.html">
        <p>Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.</p>
    </blockquote>
    <figcaption>—Aldous Huxley, <cite>Brave New World</cite></figcaption>
</figure>
```

Reply:
Good suggestion, I created an issue (https://github.com/satellytes/satellytes.com/issues/221) for it

Reply:
Thanks 👍